index.html

```diff
@@ -880,30 +886,40 @@
             In a <a>controlling user agent</a>, the <dfn for=
             "Presentation"><code>defaultRequest</code></dfn> attribute MUST
             return the <a>default presentation request</a> if any,
-            <code>null</code> otherwise. In a <a>receiving browsing
-            context</a>, it MUST return <code>null</code>.
+            <code>null</code> otherwise. On setting, the <a>default
+            presentation request</a> MUST be set to the new value.
```

Comment:
This implies that the defaultRequest can be set in a receiving browsing context.  The spec doesn't have a concrete notion of how a receiving browsing context could be used to initiate its own presentation (the "daisy chaining case") but I don't think we should exclude it.